packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -366,178 +377,6 @@ describe("Integration tests", () => {
-      });
-
-      it("Should throw expirationDateCannotBeInThePast when expiration date is in the past", async () => {
-        const expirationDateinPast = new Date(
-          currentDate.setDate(currentDate.getDate() - 3)
-        );
-
-        const updateVerifiedTenantAttributeSeed: UpdateVerifiedTenantAttributeSeed =
-          {
-            expirationDate: expirationDateinPast.toISOString(),
-          };
-
-        await addOneTenant(tenant);
-        expect(
-          tenantService.updateTenantVerifiedAttribute(
-            {
-              verifierId,
-              tenantId: tenant.id,
-              attributeId,
-              updateVerifiedTenantAttributeSeed,
-            },
-            {
-              correlationId: generateId(),
-              logger: genericLogger,
-              serviceName: "",
-              authData: getMockAuthData(),
-            }
-          )
-        ).rejects.toThrowError(
-          expirationDateCannotBeInThePast(expirationDateinPast)
-        );
-      });
-      it("Should throw verifiedAttributeNotFoundInTenant when the attribute is not verified", async () => {
-        const updatedCertifiedTenant: Tenant = {
-          ...mockTenant,
-          attributes: [{ ...getMockCertifiedTenantAttribute() }],
-          updatedAt: currentDate,
-          name: "A updatedCertifiedTenant",
-        };
-        const attributeId = updatedCertifiedTenant.attributes.map(
-          (a) => a.id
-        )[0];
-        await addOneTenant(updatedCertifiedTenant);
-        expect(
-          tenantService.updateTenantVerifiedAttribute(
-            {
-              verifierId: generateId(),
-              tenantId: updatedCertifiedTenant.id,
-              attributeId,
-              updateVerifiedTenantAttributeSeed,
-            },
-            {
-              correlationId: generateId(),
-              logger: genericLogger,
-              serviceName: "",
-              authData: getMockAuthData(),
-            }
-          )
-        ).rejects.toThrowError(
-          verifiedAttributeNotFoundInTenant(
-            updatedCertifiedTenant.id,
-            attributeId
-          )
-        );
-      });
-      it("Should throw organizationNotFoundInVerifiers when the organization is not verified", async () => {
-        await addOneTenant(tenant);
-        const verifierId = generateId();
-        expect(
-          tenantService.updateTenantVerifiedAttribute(
-            {
-              verifierId,
-              tenantId: tenant.id,
-              attributeId,
-              updateVerifiedTenantAttributeSeed,
-            },
-            {
-              correlationId: generateId(),
-              logger: genericLogger,
-              serviceName: "",
-              authData: getMockAuthData(),
-            }
-          )
-        ).rejects.toThrowError(
-          organizationNotFoundInVerifiers(verifierId, tenant.id, attributeId)
-        );
-      });
-    });
```

Comment:
What is the reason for deleting these?

Reply:
Because it was duplicated, if you look inside `integration.test` there is the same `describe` but with the addition of the `correlationId`

Reply:
Sorry, I forgot we already noticed this some time ago.